release/sbin/clear-all.sh

```diff
@@ -35,6 +35,18 @@ rm_dir() {
 if [[ ${OPENMLDB_MODE} == "standalone" ]]; then
   rm -rf standalone_db standalone_logs
 else
+  conf_file="conf/tablet.flags.template"
+  dirname=()
+  while IFS= read -r line
+  do
+    if echo "$line" | grep -q '^#'; then
+      continue
+    fi
+    if echo "$line" | grep -v "zk_root_path" | grep -q "root_path" ||
+        echo "$line" |  grep -q "openmldb_log_dir"; then
+      dirname[${#dirname[*]}]=$(echo "${line}" | awk -F '=' '{print $2}')
```

Comment:
dirname+=

Reply:
what dose it mean?

Reply:
array concat: https://stackoverflow.com/a/1951523